vars/managerPipeline.groovy

```diff
@@ -46,7 +46,14 @@ def call(Map pipelineParams) {
             string(defaultValue: "${pipelineParams.get('ip_ssh_connections', 'private')}",
                    description: 'private|public|ipv6',
                    name: 'ip_ssh_connections')
-            string(defaultValue: '', description: 'If empty - the default manager version will be taken', name: 'scylla_mgmt_repo')
+
+            string(defaultValue: '',
+                   description: 'If empty - the default manager version will be taken',
+                   name: 'scylla_mgmt_repo')
+
+            string(defaultValue: '',
+                   description: 'If empty - the latest manager version will be taken',  // TODO: apply
```

Comment:
add better description, smth. like: `link to the repository that will be used as target upgrade version of the Scylla `Manager